packages/calcite-components/src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -373,19 +373,18 @@ describe("calcite-input-date-picker", () => {
 
       it("toggles the date picker when clicked", async () => {
         let calendar = await page.find(`calcite-input-date-picker >>> .${CSS.calendarWrapper}`);
```

Comment:
Can you roll these changes back? This file now only has stylistic changes, so they are no longer needed. Applies to `globalStyles.e2e.ts` too.